docs/examples/Bfloat16Inference0.cpp

```diff
@@ -0,0 +1,8 @@
+#include <inference_engine.hpp>
+
+int main() {
+using namespace InferenceEngine;
+InferenceEngine::Core core;

+auto cpuOptimizationCapabilities = core.GetMetric("CPU", METRIC_KEY(OPTIMIZATION_CAPABILITIES)).as<std::vector<std::string>>();


```

Comment:
You may want to run `dos2unix` tool over these files to normalize line endings (I assume that `.gitattributes` is valid)